yt/geometry/coordinates/geographic_coordinates.py

```diff
@@ -281,6 +288,34 @@ def image_axis_name(self):
                 ('longitude', 'altitude'),
                 ('altitude', 'latitude'))
 
+    _data_projection = None
+    @property
+    def data_projection(self):
```

Comment:
It might be possible to call the property on the base class, then re-assign `_data_projection[self.radial_axis]` but that's not necessary.